bigstring.opam

```diff
@@ -0,0 +1,20 @@
+opam-version: "1.2"
+name: "bigstring"
+version: "0.1.1"
+authors: "Simon Cruanes <simon.cruanes.2007@m4x.org>"
+maintainer: "Simon Cruanes <simon.cruanes.2007@m4x.org>"
+tags: [ "bigstring" "bigarray" ]
+homepage: "https://github.com/c-cube/ocaml-bigstring/"
+bug-reports: "https://github.com/c-cube/ocaml-bigstring/issues"
+dev-repo: "git://github.com/c-cube/ocaml-bigstring"
+
+available: [ ocaml-version >= "4.06.0" ]
```

Comment:
why such a constraint? This makes the library unusable for anyone except people living on the bleeding edge